arcgis-ios-sdk-samples/Route & Directions/Offline routing/README.metadata.json

```diff
@@ -0,0 +1,38 @@
+{
+    "category": "Route and Directions",
```

Comment:
Does this need to be Route & Navigation to match the folder structure in the sample repo? ... or is it OK to be Route and Navigation (which is more readable).